assets/scripts/datalayer/common.js

```diff
@@ -181,9 +182,29 @@ document.shopAnalytics = {
     .on('click', shopAnalytics.event.click.registerOnCheckout, onRegisterOnCheckoutSubmit)
     .ajaxComplete(onLoad)
     .on('click', '.products .product a', onProductClick)
+    .on('click', shopAnalytics.product.elements.addToCartButton, onProductAddToCart)
```

Comment:
What about this line: https://github.com/netzstrategen/wordpress-shop-analytics/commit/939ab91fe86f01d003431a1903a685bbdba432a1#diff-942c2f9695ee8099975676219004364bR6? Isn't this necessary anymore?

Reply:
As far as I see, it should not be needed. Event click on button with class`add_to_cart_button` triggers event `adding_to_cart`:
- https://github.com/netzstrategen/gaco/blob/2d4eb3fbf3a3ccc53d79aa53f0170e1a01611402/wp-content/plugins/woocommerce/assets/js/frontend/add-to-cart.js#L13
- https://github.com/netzstrategen/gaco/blob/2d4eb3fbf3a3ccc53d79aa53f0170e1a01611402/wp-content/plugins/woocommerce/assets/js/frontend/add-to-cart.js#L23-L44